tests/blanket-options.js

```diff
@@ -0,0 +1,19 @@
+/* globals blanket, module */
+
+var options = {
+  modulePrefix: 'ember-frost-notifier',
+  filter: '//.*ember-frost-notifier/.*/',
+  antifilter: '//.*(tests|template).*/',
+  loaderExclusions: [],
+  enableCoverage: true,
+  cliOptions: {
```

Comment:
I gave this a try but getting an error.  I'll try to resolve this when I fix the other issues.